app/controllers/webhook_headers_controller.rb

```diff
@@ -0,0 +1,49 @@
+class WebhookHeadersController < ApplicationController
+  respond_to :html, :js
+
+  before_action :set_namespace
+  before_action :set_webhook
+  before_action :set_webhook_header, only: [:destroy]
```

Comment:
Since this last one is only used in the `destroy` method and both the `destroy` method and this one are trivial, it would be better to merge the code into the `destroy` method.